app/celeryapp/celery_worker.py

```diff
@@ -0,0 +1,10 @@
+from app import celeryapp, create_app
```

Comment:
a  worker is  just  a  special  function

hence  when defining workers  you could organize them in modules like `utils`

and instead of calling them worker everywhere we could maybe name the module and worker  names by what they are focused on accomplishing as a purpose